docs/advanced/memory.md

```diff
@@ -0,0 +1,17 @@
+---
+slug: /advanced/memory
+title: 💽 Memory management
+---
+
+## Memory management
+There are situations where you might need to limit the memory usage of Chroma. By default, Chroma loads the entire queried collection into its in-memory cache. To address this, we have implemented a mechanism to set a memory limit, which Chroma endeavors to adhere to as closely as possible through various cache eviction strategies. Here are some important points to note:
```

Comment:
Can we also add a note in the list below that unloading happens reactively as the user queries new collections, old ones are unloaded.